### DISCUSSION TITLE
how to run pikafish on macOS ?

### DISCUSSION THREAD

**Discussion by Youarerare** [official-pikafish/Pikafish#37] [Q&A]
I have compiled the pikafish follow the instruction on https://github.com/official-pikafish/Pikafish. After I run ./pikafish ,  there shows that: “ Pikafish dev 2023-03-14 by the Pikafish developers (see AUTHORS file) ”. how to continue with it? 
Should I get a  graphical user interface on MacOS? 
someone can help me? Thanks very much!

**Comment by PikaCat-OuO** [comment]
You can use a GUI on mac, maybe you should use liground but need to be compatible with UCI-CYCLONE, consider using mtaktikos's liground compatible version of Pikafish.

**Comment by chuotbach2020** [comment]
mtaktikos's liground liground-0.5.0-mac.zip cannot add pikafish engine
